test/test_zchunked_array.cpp

```diff
@@ -112,26 +116,6 @@ namespace xt
         EXPECT_EQ(a, b);
     }
 
-
-    TEST(zchunked_array, no_noalias_assign)
```

Comment:
Why removing this test?

Reply:
was not intended, but is added back!

Reply:
is removed. Stranegly the diff above the comment is not marked as outdated.
But here https://github.com/xtensor-stack/zarray/pull/59/files#diff-f71ddb80832dbcaa59b0a85e67f2f7227d1fc5e57ba81e073c4641601f072e38 one can see that the test is back!